using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md

```diff
@@ -2,9 +2,9 @@
 
 ## Events handled
 
-The following events are handled by the Campaign Classic extension:
+The following events are handled by the Adobe Campaign Classic extension:
 
-### Campaign Request Content   <a id="campaignclassic-content-request"></a>
+### Campaign request
```

Comment:
The type of the event is actually Campaign request content, so let's keep it the same. 
Please note that there are multiple types of request events, such as request content, request identity or request reset.